README.md

```diff
@@ -78,11 +78,11 @@ The passphrase argument is optional if `STATICRYPT_PASSWORD` is set in the envir
                    [string] [default: "/geek/staticrypt/lib/password_template.html"]
       -i, --instructions            Special instructions to display to the user.
                                                               [string] [default: ""]
-          --label-error             Error message to display on entering wrong
+          --label-error             Error message to display on entering the wrong
```

Comment:
This section of the readme is actually generated by the cli tool - the match for this line is here for example:

https://github.com/robinmoisson/staticrypt/blob/main/cli/index.js#L58

Do you mind updating the language there, regenerate the doc and copy-pasting it in the readme? I can do it otherwise